{
  "api": "search",
  "file": "/test/free/data_stream/210_rollover_failure_store.yml",
  "name": "Lazily roll over a data stream's failure store after an ingest failure",
  "origin": "yaml",
  "request": {
    "args": {
      "index": ".fs-data-stream-for-lazy-rollover-*",
      "typed_keys": true
    }
  },
  "response": {
    "headers": {
      "content-type": "application/json",
      "transfer-encoding": "chunked",
      "x-elastic-product": "Elasticsearch"
    },
    "payload": {
      "_shards": {
        "failed": 0,
        "skipped": 0,
        "successful": 2,
        "total": 2
      },
      "hits": {
        "hits": [
          {
            "_id": "vYUg_pcBRwbahH1o4zKM",
            "_index": ".fs-data-stream-for-lazy-rollover-2025.07.12-000002",
            "_score": 1,
            "_source": {
              "@timestamp": "2025-07-12T10:14:17.665Z",
              "document": {
                "index": "data-stream-for-lazy-rollover",
                "source": {
                  "@timestamp": "2020-12-12",
                  "count": "invalid value"
                }
              },
              "error": {
                "message": "error_message",
                "pipeline": "failing_pipeline",
                "pipeline_trace": [
                  "failing_pipeline"
                ],
                "processor_type": "fail",
                "stack_trace": "o.e.i.c.FailProcessorException: error_message\n\tat o.e.i.c.FailProcessor.execute(FailProcessor.java:43)\n\tat o.e.i.CompoundProcessor.innerExecute(CompoundProcessor.java:171)\n\t... 19 more\n",
                "type": "fail_processor_exception"
              }
            }
          },
          {
            "_id": "voUg_pcBRwbahH1o5DIL",
            "_index": ".fs-data-stream-for-lazy-rollover-2025.07.12-000003",
            "_score": 1,
            "_source": {
              "@timestamp": "2025-07-12T10:14:17.776Z",
              "document": {
                "index": "data-stream-for-lazy-rollover",
                "source": {
                  "@timestamp": "2020-12-12",
                  "count": 1
                }
              },
              "error": {
                "message": "error_message",
                "pipeline": "failing_pipeline",
                "pipeline_trace": [
                  "failing_pipeline"
                ],
                "processor_type": "fail",
                "stack_trace": "o.e.i.c.FailProcessorException: error_message\n\tat o.e.i.c.FailProcessor.execute(FailProcessor.java:43)\n\tat o.e.i.CompoundProcessor.innerExecute(CompoundProcessor.java:171)\n\t... 19 more\n",
                "type": "fail_processor_exception"
              }
            }
          }
        ],
        "max_score": 1,
        "total": {
          "relation": "eq",
          "value": 2
        }
      },
      "timed_out": false,
      "took": 1
    },
    "statusCode": 200
  }
}